### PR TITLE
Area or perimeter

### PR DESCRIPTION
**Pull request by mufo888**
Area or perimeter codewars problem

### CODE REVIEW

File: 8kyu/area-or-perimeter.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+const areaOrPerimeter = function(l, w) {
```
Comment (colopatiron) [inline 8kyu/area-or-perimeter.js]: Consider changing this to an arrow function.
```suggestion
const areaOrPerimeter = (l, w) => (l === w) ? l * w : 2 * (l + w);
```

**Review by colopatiron — CHANGES_REQUESTED** [review]

**Comment by colopatiron** [comment]
Left a review.

File: 8kyu/area-or-perimeter.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+You are given the `length` and `width` of a 4-sided polygon. The polygon can either be a rectangle or a square.
+If it is a square, return its area. If it is a rectangle, return its perimeter.
+
+```
+area_or_perimeter(6, 10) --> 32
+area_or_perimeter(3, 3) --> 9
+```
+
+Note: for the purposes of this kata you will assume that it is a square if its `length` and `width` are equal, otherwise it is a rectangle.
```
Comment (colopatiron) [inline 8kyu/area-or-perimeter.md:11]: LGTM

Comment (mufo888) [reply 8kyu/area-or-perimeter.md:11]: Thanks

File: 8kyu/area-or-perimeter.js (hunk 1 above)
Comment (mufo888) [reply 8kyu/area-or-perimeter.js]: This is done

**Review by colopatiron — APPROVED** [review]
Looks good to me. Go ahead and merge.

**Pull request merged**